plugins/invisible-typing/src/index.ts

```diff
@@ -28,7 +28,7 @@ export async function start(): Promise<void> {
       const globalInvisible = cfg.get("invisible", true);
       const channelWise = cfg.get("button", true) ? cfg.get("channelWise", true) : false;
       const channels = cfg.get("channels", { [channelId]: globalInvisible });
-      if (channelWise ? channels[channelId] : globalInvisible) {
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consistent fallback to global invisibility setting.

Similar to the change in Icon.tsx, this ensures that the global invisibility setting is used as a fallback when a channel's setting is undefined, maintaining consistency across the plugin.

```suggestion
if (channelWise ? (channels[channelId] !== undefined ? channels[channelId] : globalInvisible) : globalInvisible) {
```